client/landing/jetpack-cloud/style.scss

```diff
@@ -164,12 +162,55 @@
 			color: var( --color-scary-60 );
 		}
 	}
+
+	// Universal Text Styles
+
+	h2 {
+		font-size: 22px;
+		font-weight: 700;
+		line-height: 32px;
+
+		@include breakpoint( '>660px' ) {
+			font-size: 36px;
+			line-height: 40px;
+		}
+	}
+
+	h3 {
+		font-size: 16px;
+		font-weight: 700;
+		line-height: 23px;
+
+		@include breakpoint( '>660px' ) {
+			font-size: 18px;
+			line-height: 23px;
+		}
+	}
+
+	p {
+		font-size: 16px;
+		line-height: 24px;
+
+		@include breakpoint( '>660px' ) {
+			font-size: 18px;
+			line-height: 24px;
+		}
+	}
+
+	a {
+		font-size: 16px;
+		line-height: 24px;
+
+		@include breakpoint( '>660px' ) {
+			font-size: 16px;
+			line-height: 24px;
+		}
+	}
 }
```

Comment:
I think the font size of the anchor element should be an inherence from its parent.
```suggestion

```

Reply:
either way I think narrowing the scope of this PR to headings, section headings, and body text is a good idea 👍